src/ray/common/id.h

```diff
@@ -531,5 +531,13 @@ DEFINE_UNIQUE_ID(ObjectID);
 #include "id_def.h"
 
 #undef DEFINE_UNIQUE_ID
+
 }  // namespace std
+
+namespace ray {
+
+TaskID RandomTaskId();
```

Comment:
This needs to stay in the test code. People should *not* be generating random task ideas anywhere outside of test code.

Reply:
Sure I can make a `ray/common/test_util.h` (or another name) and move it there? Just let me know where to move it. But I don't think I can keep it under `ray/util` since that would make `util` can't depend on `common`.

Reply:
`ray/common/test_util.h` seems fine to me :)